app/controllers/project_groups_controller.rb

```diff
@@ -55,6 +55,8 @@ def update
   end
 
   def destroy
+    @project_group.destroy
```

Comment:
Should `ProjectGroup` clean up the join table, or did you decide to leave that around as an acts of paranoid?

Reply:
I left it around since it's a "paranoid" delete.  Seemed like eventually we might want "undo."